tutorials/26_Hybrid_Retrieval.ipynb

```diff
@@ -0,0 +1,451 @@
+    "\n",
+    "* Different retrievers use non-comparable score types, like BM25 and cosine similarity.\n",
+    "* Documents may come from single or multiple sources at the same time. There should be a way to deal with duplicates in the final ranking.\n",
+    "\n",
+    "The merge and rank of the documents from different retrievers is an open problem, Haystack offers several methods in [`JoinDocuments`](https://docs.haystack.deepset.ai/docs/join_documents), here we use the simplest, `concatenate` and we postpone the task to the ranker.\n",
+    "\n",
+    "We use a [re-ranker based on a cross-encoder](https://docs.haystack.deepset.ai/docs/ranker#sentencetransformersranker) that scores the relevancy of all candidates for the given search query.\n",
+    "For more information about the `Ranker`, check the Haystack [docs](https://docs.haystack.deepset.ai/docs/ranker)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "d_RiKspTlXRl"
+   },
+   "outputs": [],
+   "source": [
+    "from haystack.nodes import JoinDocuments, SentenceTransformersRanker\n",
+    "\n",
+    "join_documents = JoinDocuments(join_mode=\"concatenate\")\n",
+    "rerank = SentenceTransformersRanker(model_name_or_path=\"cross-encoder/ms-marco-MiniLM-L-6-v2\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "PexSrsBLlXRp"
+   },
+   "source": [
+    "## 5) Create an Hybrid Retrieval Pipeline\n",
+    "\n",
+    "With a Haystack `Pipeline`, you can connect your building blocks into a search pipeline. Under the hood, `Pipelines` are Directed Acyclic Graphs (DAGs) that you can easily customize for your own use cases. To speed things up, Haystack also comes with a few predefined `Pipelines`.\n",
+    "You can learn more about Pipelines in the [docs](https://docs.haystack.deepset.ai/docs/pipelines)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "i0XLbnAXlXRt"
+   },
+   "outputs": [],
+   "source": [
+    "from haystack.pipelines import Pipeline\n",
+    "\n",
+    "pipeline = Pipeline()\n",
+    "pipeline.add_node(component=sparse_retriever, name=\"SparseRetriever\", inputs=[\"Query\"])\n",
+    "pipeline.add_node(component=dense_retriever, name=\"DenseRetriever\", inputs=[\"Query\"])\n",
+    "pipeline.add_node(component=join_documents, name=\"JoinDocuments\", inputs=[\"SparseRetriever\", \"DenseRetriever\"])\n",
+    "pipeline.add_node(component=rerank, name=\"ReRanker\", inputs=[\"JoinDocuments\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "V3bsFkHuhHn4"
+   },
+   "source": [
+    "### Generating a Pipeline Diagram\n",
+    "\n",
+    "With any Pipeline, whether prebuilt or custom constructed, you can save a diagram showing how all the components are connected."
```

Comment:
Now that we already have the pipeline image, why not use it?
```suggestion
    "With any Pipeline, whether prebuilt or custom constructed, you can save a diagram showing how all the components are connected. For example, the hybrid pipeline should look like this:"
```
And then you can add this :
```md
    ![Hybrid pipeline graph](https://github.com/deepset-ai/haystack-tutorials/raw/main/tutorials/img/tutorial_26_pipeline_hybrid.png)
```